components/brave_rewards/browser/test/rewards_browsertest.cc

```diff
@@ -210,49 +199,39 @@ IN_PROC_BROWSER_TEST_F(
   ASSERT_EQ(tip_options, std::vector<double>({ 5, 10, 20 }));
 }
 
-// Disabled in https://github.com/brave/brave-browser/issues/10789
-IN_PROC_BROWSER_TEST_F(RewardsBrowserTest, DISABLED_NotVerifiedWallet) {
+IN_PROC_BROWSER_TEST_F(RewardsBrowserTest, NotVerifiedWallet) {
+  rewards_browsertest_util::CreateWallet(rewards_service_);
+  context_helper_->LoadRewardsPage();
   contribution_->AddBalance(promotion_->ClaimPromotionViaCode());
   contribution_->IsBalanceCorrect();
 
-  // Click on verify button
   rewards_browsertest_util::WaitForElementThenClick(
-      contents(),
-      "#verify-wallet-button");
+      contents(), "[data-test-id=verify-rewards-button]");
 
-  // Click on verify button in on boarding
   rewards_browsertest_util::WaitForElementThenClick(
-      contents(),
-      "#on-boarding-verify-button");
+      contents(), "[data-test-id=connect-continue-button]");
+
+  rewards_browsertest_util::WaitForElementThenClick(
+      contents(), "[data-test-id=connect-provider-button]");
 
   // Check if we are redirected to uphold
-  {
-    const GURL current_url = contents()->GetURL();
-    ASSERT_TRUE(base::StartsWith(
-        current_url.spec(),
-        ledger::uphold::GetUrl() + "/authorize/",
-        base::CompareCase::INSENSITIVE_ASCII));
-  }
+  WaitForNavigation(ledger::uphold::GetUrl() + "/authorize/");
+
+  response_->SetVerifiedWallet(true);
 
   // Fake successful authentication
   ui_test_utils::NavigateToURLBlockUntilNavigationsComplete(
         browser(),
         uphold_auth_url(), 1);
 
-  // Check if we are redirected to KYC page
-  {
-    const GURL current_url = contents()->GetURL();
-    ASSERT_TRUE(base::StartsWith(
-        current_url.spec(),
-        ledger::uphold::GetUrl() + "/signup/step2",
```

Comment:
We no longer support this flow. Shouldn't this test be removed instead?

Reply:
I modified the test so that it checks to see that the rewards page transitions to the "verified" UX instead of redirecting the user to the KYC URL. I think it's still probably useful for covering the normal flow?

Reply:
Yeah, right. I thought the normal flow is covered somewhere else already. 👍🏻